trainer.py

```diff
@@ -63,7 +65,7 @@
 
         # Initialize model, loss, optimizer
         #model = DesneNet121().to(device)
-        model = ResNet50(num_classes=1, channels=1).to(device)
+        model = ResNet50(num_classes=2, channels=1).to(device)
```

Comment:
we will have to adapt also the trainer to adapt the number of classes depending on the number of labels returned by the data loader, isnt it ? ([ref todo2](https://github.com/mvivopas/RSNA-2023/issues/8#issuecomment-1724894711))

Reply:
To address in another iteration